docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -951,10 +956,139 @@ Step 6: First Boot
    Replace ``YOUR_USERNAME`` with your desired username::
 
      username=YOUR_USERNAME
+     
+   Choose one of the following:
+     
+   - Unencrypted homedir or whole-disk encryption
+     
+       ::
+
+         zfs create rpool/home/$username
+         adduser $username
+       
+   - Encrypted homdir per user, automatic decryption on login
+     
+       ::
+       
+         zfs create rpool/home/$username -o encryption=on -o keyformat=passphrase -o keylocation=prompt -o canmount=noauto -o org.openzfs.systemd:ignore=on
+         zfs mount rpool/home/$username
+         adduser $username
+         
+       **Note**: Use the same strong password for ZFS encryption and user password. Please note: After a breach of your password changing the ZFS password does not restore protection.
+       
+       Tell PAM to unlock the dataset key on login::
+       
+         printf "auth [success=1 default=ignore] pam_succeed_if.so service = sudo quiet\n\
```

Comment:
I'd probably make this use `vi` as the example. I've tried to make the longer stuff just invoke the editor. I think I used to use multi-line heredocs and people didn't love that.